app/src/main/java/com/duckduckgo/app/browser/shortcut/ShortcutReceiver.kt

```diff
@@ -54,9 +54,11 @@ class ShortcutReceiver @Inject constructor(
         }
 
         GlobalScope.launch(dispatcher.io()) {
-            if (ctaDao.exists(CtaId.USE_OUR_APP) && originUrl == USE_OUR_APP_SHORTCUT_URL) {
-                userEventsStore.registerUserEvent(UserEventKey.USE_OUR_APP_SHORTCUT_ADDED)
+            if (useOurAppDetector.isUseOurAppUrl(originUrl)) {
                 pixel.fire(Pixel.PixelName.USE_OUR_APP_SHORTCUT_ADDED)
+                if (variantManager.getVariant().hasFeature(VariantManager.VariantFeature.InAppUsage)) {
+                    userEventsStore.registerUserEvent(UserEventKey.USE_OUR_APP_SHORTCUT_ADDED)
```

Comment:
Only happens for the experimental variant to avoid showing the fireproof dialog in the other variants by accident if a user adds the use our app shortcut.